src/main/java/frc/robot/Constants.java

```diff
@@ -101,6 +101,44 @@ public static final class CanalConstants {
     public static final int LASER_MAX_DISTANCE = 180;
   }
 
+  public enum ElevatorSetpoint {
+    elevatorHover(6.19),
```

Comment:
```suggestion
    HOVER(6.19),
```

These are constants, so we cans caps them. No need for elevator prefix since when you use it it'll be `ElevatorSetpoint.HOVER`